src/components/cartcompareitems.jsx

```diff
@@ -0,0 +1,722 @@
+import { itemLookup, cortexFetchItemLookupForm } from '../utils/CortexLookup';
+import {
+  isAnalyticsConfigured, trackAddItemAnalytics, trackAddImpression, setAddAnalytics, sendAddToCartAnalytics, setDetailAnalytics,
+} from '../utils/Analytics';
+import imgPlaceholder from '../images/img-placeholder.png';
+// import ProductRecommendationsDisplayMain from './productrecommendations.main';
+import IndiRecommendationsDisplayMain from './indirecommendations.main';
+import BundleConstituentsDisplayMain from './bundleconstituents.main';
+import { cortexFetch } from '../utils/Cortex';
+
+import './cartcompareitems.less';
+
+const Config = require('Config');
+
+// Array of zoom parameters to pass to Cortex
+const zoomArray = [
+  'availability',
+  'addtocartform',
+  'addtowishlistform',
+  'price',
+  'rate',
+  'definition',
+  'definition:assets:element',
+  'definition:options:element',
+  'definition:options:element:value',
+  'definition:options:element:selector:choice',
+  'definition:options:element:selector:chosen',
+  'definition:options:element:selector:choice:description',
+  'definition:options:element:selector:chosen:description',
+  'definition:options:element:selector:choice:selector',
+  'definition:options:element:selector:chosen:selector',
+  'definition:options:element:selector:choice:selectaction',
+  'definition:options:element:selector:chosen:selectaction',
+  'definition:components',
+  'definition:components:element',
+  'definition:components:element:code',
+  'definition:components:element:standaloneitem',
+  'definition:components:element:standaloneitem:code',
+  'definition:components:element:standaloneitem:definition',
+  'definition:components:element:standaloneitem:availability',
+  'recommendations',
+  'recommendations:crosssell',
+  'recommendations:recommendation',
+  'recommendations:replacement',
+  'recommendations:upsell',
+  'recommendations:warranty',
+  'recommendations:crosssell:element:code',
+  'recommendations:recommendation:element:code',
+  'recommendations:replacement:element:code',
+  'recommendations:upsell:element:code',
+  'recommendations:warranty:element:code',
+  'recommendations:crosssell:element:definition',
+  'recommendations:recommendation:element:definition',
+  'recommendations:replacement:element:definition',
+  'recommendations:upsell:element:definition',
+  'recommendations:warranty:element:definition',
+  'recommendations:crosssell:element:price',
+  'recommendations:recommendation:element:price',
+  'recommendations:replacement:element:price',
+  'recommendations:upsell:element:price',
+  'recommendations:warranty:element:price',
+  'recommendations:crosssell:element:availability',
+  'recommendations:recommendation:element:availability',
+  'recommendations:replacement:element:availability',
+  'recommendations:upsell:element:availability',
+  'recommendations:warranty:element:availability',
+  'code',
+];
+
+class Cartcompareitems extends React.Component {
```

Comment:
Naming convention. Can we follow our traditional conventions?
`CartCompareItems `